### PR TITLE
Drag-and-drop of dp files

### LINKED ISSUE #2940: Drag and drop of deployment packages
Currently the drag-and-drop feature for installing deployment packages is limited to the Eclipse marketplace. So, only a link to the marketplace is allowed in to the packages tab.

IMHO, a useful feature could be the upload of .dp files from the local filesystem with drag-and-drop.

### PR DESCRIPTION
**Pull request by pierantoniomerlino**
This PR adds the possibility to drag-and-drop deployment packages directly from the local filesystem.

**Related Issue:** This PR fixes/closes #2940 

Thanks to @nicolatimeus for the contribution!

### CODE REVIEW

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/util/DropSupport.java [hunk 1]
```diff
@@ -12,7 +12,10 @@
 
 package org.eclipse.kura.web.client.util;
 
+import java.util.List;
```
Comment (LeoNerdoG) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/util/DropSupport.java:15]: Please update header of the file.

**Review by LeoNerdoG — CHANGES_REQUESTED** [review]
Hey @pierantoniomerlino,
I have checked this PR and I have two comments:
1. please fix the header in DropSupport.java file
2. if user selects multiple packages from desktop and drops them in kura, Kura asks user if he wants to install the package (only one package) - if you say yes, nothing happens. Is this something we can ignore or should be fixed. 
All other scenarios are OK.

**Comment by MMaiero** [comment]
I believe it works great!
I have tested the corner case with multiple dps selected. In my case it actually installed one of them.
But yes, we should take care of that case, in order to prevent issues. Can we check the number of files selected? If so, we may want to throw an error due to the fact that we support just one dp at a time.

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/packages/PackagesPanelUi.java [hunk 2]
```diff
@@ -606,22 +611,53 @@ private boolean isEclipseMarketplaceUrl(String url) {
...
+                    event.stopPropagation();
                     PackagesPanelUi.this.packagesDropzone.removeStyleName(DROPZONE_ACTIVE_STYLE_NAME);
-                    if (isEclipseMarketplaceUrl(url)) {
-                        PackagesPanelUi.this.confirmDialog.show(MSGS.packagesMarketplaceInstallConfirmMessage(),
-                                () -> eclipseMarketplaceInstall(url));
+
+                    final List<File> droppedFiles = event.getFiles();
+
+                    if (!droppedFiles.isEmpty()) {
+
```
Comment (MMaiero) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/packages/PackagesPanelUi.java]: maybe, for readability, it is better to move the content to a specific method.

**Review by MMaiero — APPROVED** [review]
I made a small change in the message when multiple dp files are provided. @pierantoniomerlino can you have a look at it? Other than that for me it can be merged.

**Comment by pierantoniomerlino** [comment]
@MMaiero For me is ok!

**Pull request merged**